Formula/brew-pip.rb

```diff
@@ -9,4 +9,8 @@ class BrewPip < Formula
   def install
     bin.install "bin/brew-pip"
   end
+
+  test do
+    system "#{bin}/brew-pip", "help"
```

Comment:
@bfontaine Given there's no test here and `brew-gem` uses a similar format I think this is fine for now 👍